client/src/pages/Paste.js

```diff
@@ -37,6 +40,11 @@ function Paste({ match }) {
             {new Date(paste.createdAt).toDateString()}
           </CreatedAt>
           <Author name={paste.author} />
+          <VoteButton
+            pastaPoints={voteCount}
+            onVoteButtonClick={doSetVote}
+            disabled={loading || clicked}
+          ></VoteButton>
```

Comment:
This is a void function
```suggestion
          />
```